homeassistant/components/weather/meteo_france.py

```diff
@@ -0,0 +1,145 @@
+"""
+Support for Meteo france weather service.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/weather.meteo_france/
+"""
+import logging
+from datetime import datetime, timedelta
+
+import voluptuous as vol
+
+from homeassistant.components.weather import (
+    WeatherEntity, PLATFORM_SCHEMA, ATTR_FORECAST_CONDITION,
+    ATTR_FORECAST_TEMP, ATTR_FORECAST_TEMP_LOW, ATTR_FORECAST_TIME)
+from homeassistant.const import TEMP_CELSIUS
+from homeassistant.helpers import config_validation as cv
+# Reuse data and API logic from the sensor implementation
+from homeassistant.components.sensor.meteo_france import \
+    MeteoFranceUpdater, CONF_POSTAL_CODE, CONF_ATTRIBUTION
```

Comment:
Unfortunately we allowed this in the past...I'm sorry, today you need to create a `meteo_france` component and make the `weather` and the `sensor` platform depend on the component which contain the logic to retrieve the data.

Reply:
Thanks for the review
I will update the PR in a few days with an appropriate solution